packages/cozy-client/src/models/paper.js

```diff
@@ -134,17 +139,13 @@ export const computeExpirationNoticeDate = file => {
 /**
  * @param {IOCozyFile} file - io.cozy.files document
  * @returns {string | null} Expiration notice link
- * @description Computes et returns the expiration notice link of the given file, or null if it has none or it is not expiring
+ * @description Computes et returns the expiration notice link of the given file, or null if it has none
```

Comment:
s/et/and ;)

Reply:
Bien vu, on l'a donc manqué au moins à 2 reprises 👓